cdk/lib/newsletters-tool.ts

```diff
@@ -146,7 +151,11 @@ EOL`,
 		const ec2AppApi = new GuNodeApp(this, {
 			access: { scope: AccessScope.PUBLIC },
 			certificateProps: { domainName: domainNameApi },
-			monitoringConfiguration: { noMonitoring: true },
+			monitoringConfiguration: {
+				http5xxAlarm: { tolerated5xxPercentage: 5 },
+				snsTopicName: alarmsTopicName,
+				unhealthyInstancesAlarm: true,
+			},
```

Comment:
Not as complex as I expected.

Reply:
DevX have made our lives simpler here. 😄